docs/source/catalog.rst

```diff
@@ -134,11 +134,11 @@ A Catalog can be loaded from a YAML file on the local filesystem by creating a C
 
 Then sources can be listed::
 
-    cat.list()
+    list(cat)
 
 and data sources are loaded with ``get()``::
 
-    data = cat.get('entry1_part', part='1')
+    data = cat['entry_part1'].get(part='1')
```

Comment:
Does this work? `cat.entry_part1(part='1')` is probably the simplest form.

Reply:
That's even nicer, updated.